assets/js/src/newsletters/send/re_engagement.tsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import MailPoet from 'mailpoet';
+import { assoc, find, map } from 'lodash/fp';
+
+import { Scheduling } from '../types/re_engagement/scheduling';
+import GATrackingField from './ga_tracking';
+import SenderField from './sender_address_field';
+
+interface ReengagementWindow extends Window {
+  settings: {
+    deactivate_subscriber_after_inactive_days: string;
+  }
+}
+
+declare let window: ReengagementWindow;
```

Comment:
We could [reuse the type definition of settings object](https://github.com/mailpoet/mailpoet/blob/2860db41e588400d8754dc6273fa7839b4f648d0/assets/js/src/settings/store/types.ts#L1-L137) and also pass the settings object via MailPoet module as we do for other static values.